src/EnergyPlus/DataZoneEquipment.cc

```diff
@@ -492,6 +496,7 @@ void GetZoneEquipmentData(EnergyPlusData &state)
             auto const &objectFields = instance.value();
             auto &thisZeqSplitter = state.dataZoneEquip->zoneEquipSplitter[zeqSplitterNum];
             thisZeqSplitter.Name = UtilityRoutines::makeUPPER(instance.key());
+            thisZeqSplitter.spaceEquipType = DataLoopNode::ConnectionObjectType::SpaceHVACZoneEquipmentSplitter;
```

Comment:
spaceEquipType, huh?  OK!

Reply:
Because of the shared `size` function, the splitter and mixer objects needs to know what they are, so added this `spaceEquipType` variable to keep track.